### PR TITLE
Newsletters: add 252 (2023-05-24)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic links @harding 
- [x] Bitcoin Core 27021 @harding 
- [x] Waiting for confirmation 2 @glozow 
- [x] Client/Services @bitschmidty

### CODE REVIEW

File: _posts/en/newsletters/2023-05-24-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    agree that their nodes all validated those blocks successfully.
+
+    To minimize the complexity of their proof, they use a version of
+    [utreexo][topic utreexo] with a hash function optimized for their
+    system.  They separately suggest that combining their proof with a
+    utreexo client will allow that client to begin operating as a full
+    node almost immediately after downloading a very small amount of
+    data.
+
+    Regarding the usability of their prototytes, they write that "we
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-05-24-newsletter.md:46]:
```suggestion
    Regarding the usability of their prototypes, they write that "we
```

File: _posts/en/newsletters/2023-05-24-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    also describe a protocol that can be used for a
+    client-side-validation token protocol similar to that used for LL's
+    Taproot Assets and some uses of RGB (see Newsletters [#195][news195
+    taro] and [#247][news247 rgb]).  When Alice transfers to Bob some
+    amount of a token, Bob needs to verify the history of every previous
+    transfer of those specific tokens back to when they were created.
+    In an ideal scenario, that history grows linearly with the number of
+    transfers.  But if Bob wants to pay Carol an amount higher than he
+    received from Alice, he will need to combine the amount of tokens he
+    received from Alice with an amount of tokens he received in a
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-05-24-newsletter.md:64]: (IIUC)
```suggestion
    received from Alice, he will need to combine some tokens he
    received from Alice with some tokens he received in a
```

File: _posts/en/newsletters/2023-05-24-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    history going back through Alice and also the history of Bob's other
+    tokens.  This is called a merge.  If merges happen often, the size
+    of the history that needs to be verified approaches the size of the
+    history of every transfer of that token between any users.  In
+    comparative terms, in Bitcoin every full node verifies every
+    transaction made by every user; in token protocols using client-side
+    validation, that's not strictly required but eventually becomes
+    effectively the case if merges are common.
+
+    That means a protocol which can compress the state of Bitcoin can
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-05-24-newsletter.md:75]:
```suggestion
    That means a protocol that can compress the state of Bitcoin can
```

File: _posts/en/newsletters/2023-05-24-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27021][] Implement Mini version of BlockAssembler to calculate mining scores FIXME:harding (hoping glozow or Xekyo will volunteer, though)
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-05-24-newsletter.md:139]: If neither of them is available, I could help out. (I don't know this PR as well as they do, but I know it fairly well.)

Comment (Xekyo) [reply _posts/en/newsletters/2023-05-24-newsletter.md:139]: I can give it a spin.

File: _posts/en/newsletters/2023-05-24-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+- [Core Lightning 23.05rc2][] is a release candidate for the next
+  version of this LN implementation.
+
+- [Bitcoin Core 23.2rc1][] is a release candidate for a maintenance
+  release of the previous major version of Bitcoin Core.
```
Comment (Xekyo) [inline _posts/en/newsletters/2023-05-24-newsletter.md:121]: Is meanwhile released: https://github.com/bitcoin/bitcoin/releases/tag/v23.2

File: _posts/en/newsletters/2023-05-24-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+
+- [Bitcoin Core 24.1rc3][] is a release candidate for a maintenance
+  release of the current version of Bitcoin Core.
```
Comment (Xekyo) [inline _posts/en/newsletters/2023-05-24-newsletter.md:124]: Meanwhile released: https://github.com/bitcoin/bitcoin/releases/tag/v24.1

**Review by Xekyo — COMMENTED** [review]
LGTM so far

**Comment by harding** [comment]
@Xekyo I push a substantive edit to your PR description.  Please feel free to revert or edit, of course.

**Review by Xekyo — COMMENTED** [review]
Looks great, thanks for the polish!

Reviewed the complete text, no other comments.

**Comment by glozow** [comment]
> @glozow pushed some edits to your section (rather than leaving comments) in the interest of saving time. Please feel free to revert. https://github.com/bitcoinops/bitcoinops.github.io/commit/f82617a8cf4ee7d91c220c63eef9c1c176ee3f1e

Makes perfect sense to me, thank you :)

**Comment by harding** [comment]
Pushed edits for reviews (thanks!).  Added lede, updated releases/RCs, and add topic entries.  Reviewed @glozow @Xekyo @bitschmidty excellent additions, thanks!  Thanks everyone!

File: _includes/specials/policy/en/02-cache-utility.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+Consensus rules limit the space used by transactions in each block.
+This limit keeps block propagation times low relative to the block
+interval, reducing the risk of stale blocks. It also helps restrict
+the growth of the block chain and UTXO set, both of which directly
+contribute to the cost of bootstrapping and maintaining a full node.
+
+As such, as part of their role as a cache of unconfirmed transactions,
+mempools also facilitate a public auction for inelastic block space:
+when functioning properly, the auction operates on free market
```
Comment (LarryRuane) [inline _includes/specials/policy/en/02-cache-utility.md:25]:
```suggestion
when functioning properly, the auction operates on free-market
```

File: _includes/specials/policy/en/02-cache-utility.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+incentive-compatible transactions.
+
+While there are no consensus rules requiring transactions to pay fees,
+fees and feerate play an important role in the Bitcoin network as a
+"fair" way to resolve competition for block space.  Miners use feerate
+to assess acceptance, eviction, and conflicts, while non-mining nodes
+mirror those behaviors in order to maximize the utility of their
+mempools.
+
+The scarcity of blockspace exerts a downward pressure on the size of
```
Comment (LarryRuane) [inline _includes/specials/policy/en/02-cache-utility.md:75]:
```suggestion
The scarcity of blockspace exerts downward pressure on the size of
```

File: _posts/en/newsletters/2023-05-24-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,256 @@
...
+    history of every transfer of that token between any users.  In
+    comparative terms, in Bitcoin every full node verifies every
+    transaction made by every user; in token protocols using client-side
+    validation, that's not strictly required but eventually becomes
+    effectively the case if merges are common.
+
+    That means a protocol that can compress the state of Bitcoin can
+    also be adapted to compress the state of a token's history, even one
+    where merges are common.  The authors describe how they would
+    accomplish that.  Their goal is to produce a proof that each
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-05-24-newsletter.md:83]:
```suggestion
    accomplish that.  Their goal is to develop an algorithm that can produce a proof that each
```

File: _posts/en/newsletters/2023-05-24-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,256 @@
...
+- **Trezor Suite adds coinjoin support:**
+  The Trezor Suite software [announced][trezor blog] support for
+  [coinjoins][topic coinjoin] that use the zkSNACKs coinjoin coordinator.
+
+- **Lightning Loop defaults to MuSig2:**
+  [Lightning Loop][news53 loop] now uses [MuSig2][topic musig] as the default
+  swap protocol resulting in lower fees and better privacy.
+
+- **Mutinynet announces new signet for testing:**
+  [Mutinynet][mutinynet blog] is a custom signet with 30 second block times that
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-05-24-newsletter.md:140]:
```suggestion
  [Mutinynet][mutinynet blog] is a custom signet with 30-second block times that
```

**Review by bitschmidty — COMMENTED** [review]
Took a couple of @LarryRuane's suggestions, fixed a topic entry, made a small change to the news section, and another to the "Waiting for confirmation" section

**Pull request merged**